test/manual/README.md

```diff
@@ -75,6 +75,10 @@ go run cmd/cli/main.go attestation validate '<attesation>' --report_data '<repor
 go run cmd/cli/main.go algo test/manual/algo/lin_reg.bin <private_key_file_path>
 # 2023/09/21 10:43:53 Uploading algorithm binary: test/manual/algo/lin_reg.bin
 
+# In order to run the Docker image, run the CLI program with the algorithm docker option
+go run ./cmd/cli/main.go algo -a docker -d "python3 /cocos/lin_reg.py" <path_to_docker_image.tar> <private_key_file_path>
```

Comment:
please add the docker file to test/algo for the example used, also add docs there on how to build and output the docker image

Reply:
I have added an README file for the docker.